pkg/repo/mysql/services_integration_test.go

```diff
@@ -0,0 +1,254 @@
+package mysql
+
+import (
+	"context"
+	"log"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/google/uuid"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/repo"
+	"go.expect.digital/translate/pkg/tracer"
+)
+
+var repository *Repo
+
+func TestMain(m *testing.M) {
+	ctx := context.Background()
+
+	viper.SetEnvPrefix("translate")
+	viper.SetEnvKeyReplacer(strings.NewReplacer(".", "_"))
+
+	viper.AutomaticEnv()
+
+	tp, err := tracer.TracerProvider()
+	if err != nil {
+		log.Panicf("set tracer provider: %v", err)
+	}
+
+	repository, err = NewRepo(WithDefaultDB(ctx))
+	if err != nil {
+		log.Panicf("create new repo: %v", err)
+	}
+
+	code := m.Run()
+
+	repository.db.Close()
```

Comment:
```suggestion
	defer repository.db.Close()
	
	code := m.Run()
```

Reply:
The `defer` keyword does not work with `os.Exit()` because it causes the program to end abruptly.